intranet/apps/eighth/views/admin/groups.py

```diff
@@ -416,7 +418,6 @@ def done(self, form_list, **kwargs):
 
 
 def eighth_admin_signup_group_action(request, group_id, schact_id):
-
     try:
         scheduled_activity = EighthScheduledActivity.objects.get(id=schact_id)
```

Comment:
I think this is a bit out-of-scope, but, if we are working on this, we should probably convert this to `get_object_or_404`.